tests/network/vmi_slirp_interface.go

```diff
@@ -194,7 +194,7 @@ var _ = SIGDescribe("Slirp Networking", func() {
 		})
 		It("should reject VMIs with default interface slirp when it's not permitted", func() {
 			var t int64 = 0
-			vmi := tests.NewRandomVMIWithNS(util.NamespaceTestDefault)
+			vmi := tests.NewRandomVMI()
```

Comment:
Maybe we could transform this to use libvmi and don't use the util functions...

Reply:
I believe we could and we should - but not in this PR, please.
This commit is related to the PR only in that it drops a usage of an "evil" public constant (`util.NamespaceTestDefault`) but if you insist, I would kick it out of the PR.

Reply:
I should start indicating when I require a change. This is not one of those... I apologize